assets/src/edit-story/components/header/buttons.js

```diff
@@ -134,28 +134,18 @@ function PreviewButton() {
     evt.preventDefault();
   };
 
+  const onDialogClose = () => setPreviewLinkToOpenViaDialog(null);
```

Comment:
Should this be in a `useCallback`?

Reply:
FWIW, `useCallback` is not something that always makes sense. See https://kentcdodds.com/blog/usememo-and-usecallback for example.